src/compactor.rs

```diff
@@ -242,10 +242,17 @@ impl CompactorOrchestrator {
             .filter_map(|s| s.maybe_unwrap_sorted_run())
             .filter_map(|id| srs_by_id.get(&id).map(|t| (*t).clone()))
             .collect();
+        let is_last_sorted_run = match db_state.compacted.last() {
+            Some(last_run) => last_run.id == compaction.destination,
+            None => false,
+        };
         self.executor.start_compaction(CompactionJob {
             destination: compaction.destination,
             ssts,
             sorted_runs,
+            // FIXME: Check if it's the last run
```

Comment:
I think this is correct now? (Is destination = id of the last compacted run the correct thing to check?)